x-pack/plugins/apm/public/components/alerting/rule_types/transaction_error_rate_rule_type/index.tsx

```diff
@@ -147,10 +164,44 @@ export function TransactionErrorRateRuleType(props: Props) {
     />
   );
 
+  const groupAlertsBy = (
```

Comment:
this component looks almost similar to all rules. Can we export it and reuse it?

Reply:
The `helpText` is slightly different for Error count rule as it does not have Transaction Type, that's why I didn't reuse the component.